src/assets/theming/_base.scss

```diff
@@ -4,154 +4,108 @@
 }
 
 html {
-  background-color: $bg-clr;
-  color: $base-clr;
-  font-family: $main-font;
   font-size: 50%;
   scroll-behavior: smooth;
 
   /* Better Font Rendering =========== */
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 
-
   @include breakpoint-md {
     font-size: 62.5%;
   }
 }
 
 body {
   box-sizing: border-box;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
   font-size: 2.25rem;
   height: 100vh;
   line-height: 140%;
   min-width: 320px;
-  margin: 0 5em 0 1.5em;
+  a {
+    text-decoration: none;
+    color: inherit;
+  }
 
   @include breakpoint-md {
     font-size: 1.8rem;
   }
-
-  footer {
-    justify-self: flex-end;
-  }
-}
-
-h1, h2, h3, h4, h5, h6 {
-  font-family: $accent-font;
-  line-height: 140%;
-  margin: 0 0 0.7em 0;
-}
-
-h1 {
-  font-size: 4rem;
-}
-
-a {
-  color: $link-clr;
-  transition: color 400ms;
-
-  &:hover {
-    color: $link-hvr-clr;
-  }
 }
 
-p {
-  max-width: 600px;
-  + p {
-    margin-top: 1.5rem;
-  } 
-}
-
-nav {
-  ul {
-    list-style: none;
+header {
+  background-color: $orange;
+  display: flex;
+  justify-content: space-between;
+  height: 4em;
+
+  div {
+    border: solid 1px white;
+    margin-left: 2.5em;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    font-size: 3rem;
+    width: 100px;
   }
-}
 
-main {
-  flex: 0 1 100%;
-  margin: 1em 0 2em 0;
+  nav {
+    border: solid 1px black;
+    margin-right: 2.5em;
+    font-size: 3rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    color: $purple;
+    font-family: $display-font;
 
-  &:not(nav) {
     ul {
-      margin: .7em 0 .7em 1em;
+      list-style: none;
     }
-  }
 
-  nav {
-    span {
-      font-size: 2.2rem;
-      font-weight: 600;
-    }
-  }
+    li {
+      display: inline-block;
+      margin-left: 2em;
 
-  nav ul {
-    margin: .2em 0 2em 0;
-  }
-}
+      a:hover {
+        text-decoration: underline;
+      }
 
-header,
-main,
-footer {
-  max-width: 1440px;
+      a:active {
+        text-decoration: underline;
+      }
+    }
+  }
 }
 
-header,
 footer {
-  padding: 2em 0 .7em 0;
-}
-
-header {
+  font-family: $display-font;
+  background-color: $orange;
+  height: 4em;
+  position: absolute;
```

Comment:
Position absolute is probably not the best way to make sure the footer stays at the bottom. If your body is shorter than the screen is long, it will look like it's floating. Or are you trying to use that bootstrap sticky footer method where you have like a buffer zone for your footer that pushes it down?

If not, you could go for using flexbox with direction set to column on the body and then make sure the footer sticks to the bottom.

Reply:
That's a fait point, I have already changed it to flexbox. But then when I make it smaller why do I have the white backgound instead of the end of page? 
<img width="893" alt="Screenshot 2022-08-18 at 19 37 43" src="https://user-images.githubusercontent.com/67103707/185459574-1cce6c69-3d08-48e1-b0c9-a958c992fd86.png">